var/docs/docker.md

```diff
@@ -0,0 +1,140 @@
+# Docker
+
+ * [Developer docker](#developing-in-a-docker) (apache, sqlite)
+ * [Production docker](#production-docker-compose) (docker-compose, mariadb, nginx, php-fpm)
+ * [Tests](#tests)
+
+## Developing in a docker
+
+The developer docker manages the resources required to run a checkout of Kimai against locally hosted files.  The DB data is stored in sqlite database.  File edits against the local files are reflected in the running docker.
+
+### Requirements
+
+#### Docker
+
+Follow the follwing links to install docker on your OS.
+
+ * Debian: https://www.digitalocean.com/community/tutorials/how-to-install-and-use-docker-on-debian-9
+ * Fedora: https://www.techrepublic.com/article/how-to-install-docker-on-fedora-25/
+ * Ubuntu: https://www.digitalocean.com/community/tutorials/how-to-install-and-use-docker-on-ubuntu-18-04
+ * Windows: https://docs.docker.com/docker-for-windows/install/
+
+### Building the development docker
+
+You can change the docker tag if you want. Remeber the dot at the end of the command.
```

Comment:
Talking of tags... Do you want me to create a kimai repo at dockerhub?  Then we tag as kimai/kimai2:XXX and push it.

I can make you (any any other maintainers admins on it and it will allow anyone to pull and run images.  It would link back to your github repo.